x/staging/subspaces/client/cli/query.go

```diff
@@ -0,0 +1,55 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/desmos-labs/desmos/x/staging/subspaces/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the command allowing to perform queries
+func GetQueryCmd() *cobra.Command {
+	subspaceQuerycmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the subspaces module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	subspaceQuerycmd.AddCommand(
+		GetCmdQuerySubspace(),
+	)
+	return subspaceQuerycmd
+}
+
+// GetCmdQuerySubspace returns the command to query the subspace with the given id
+func GetCmdQuerySubspace() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "subspace [id]",
+		Short: "Retrieve the subspace with the given [id] alongside its admins and the users that cant post in it",
```

Comment:
No need to specify all the returned data, as it might change in the future and we don't want to keep updating the description
```suggestion
		Short: "Get the subspace with the given [id]",
```